libebpfdiscoveryshared/headers/ebpfdiscoveryshared/Constants.h

```diff
@@ -26,3 +26,5 @@
 #define DISCOVERY_LOG_MAX_FORMAT_LENGTH 128
 #define DISCOVERY_LOG_MAX_MESSAGE_LENGTH DISCOVERY_LOG_MAX_FORMAT_LENGTH + 32
 #define DISCOVERY_LOG_MAX_ARGS_COUNT 8
+
+#define MAX_IOVLEN 3
```

Comment:
Why only 3? On Ubuntu 22 I'm getting:
```
$ getconf UIO_MAXIOV
1024
```

The maximum number of struct iovec elements that can be used in a single recvmsg() or sendmsg() call is determined by the UIO_MAXIOV constant, which is defined in the <linux/uio.h> header file.

Reply:
It was just stipulated, based on `pixie` – as BPF loops have to be artificially restricted. I will change the limit to `1024` and see if it breaks anything.